src/http/nodegen/request-worker/process.ts

```diff
@@ -16,10 +16,12 @@ const initStaticPool = () => {
     new Worker(
       `${process.cwd()}/build/src/http/nodegen/request-worker/thread.js`
     )
-      .once('error', () => {
+      .once('error', (e: any) => {
+        console.trace('worker error', e);
         workers[index] = startWorker(index);
```

Comment:
@johndcarmichael 
I'm wondering about this too - if the worker errors or exits, we just restart it, but should we be clearing the old worker?
Something like 
```typescript
.once('error', (e: any) => {
  console.trace('worker error', e);
  workers[index].end();
  workers[index] = startWorker(index);
```
Seems like at least this way it won't crash out the memory, but just spam the logs until you start paying attention